pipeline/topic_modelling/controller.py

```diff
@@ -160,7 +165,7 @@ def remove_propers_POS(s):
 
 
 def preprocessor(s):
-    s = remove_propers_POS(s)
+    #s = remove_propers_POS(s)
```

Comment:
@mkao006 this is a 10X+ improvement (from hours to 4') 
It makes sense since we're doing POS tagging for each article and it's a non-trivial operation.

It's not worth the extra computations since proper nouns are unlikely to bias the sentiment.